src/Workspaces/Remote/ServiceHub/Services/ServiceHubServiceBase.cs

```diff
@@ -60,10 +61,10 @@ protected RoslynServices RoslynServices
             }
         }
 
-        protected Task<Solution> GetSolutionAsync()
+        protected Task<Solution> GetSolutionAsync(OptionSet options = null)
```

Comment:
right now, option is a bit weird unfortunately until option becomes part of snapshot. how about adding new API GetSolutionWithSpecificOptionsAsync